src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerLimits.cs

```diff
@@ -108,5 +111,27 @@ public int MaxRequestHeaderCount
                 _maxRequestHeaderCount = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the keep-alive timeout, in seconds.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to 5 seconds.
+        /// </remarks>
+        public int KeepAliveTimeout
```

Comment:
But could lead to false expectations i.e. < 1 second wouldn't work, also e.g. 2.5 wouldn't be much different than 2 or 3 (good deal of "error" as to when timeout is checked).